openverse_catalog/dags/common/popularity/sql.py

```diff
@@ -129,6 +137,7 @@ def create_media_popularity_metrics(
 
 def update_media_popularity_metrics(
     postgres_conn_id,
+    task,
```

Comment:
This will cause an error in the `update_media_popularity_metrics_table` step of the data refresh, because it's passing `postgres_conn_id` and `media_type` in as _args_ rather than kwargs ([here](https://github.com/WordPress/openverse-catalog/blob/main/openverse_catalog/dags/data_refresh/refresh_popularity_metrics_task_factory.py#L44)). So you'll get this error when trying to unpack:

`ValueError: The key 'task' in args is a part of kwargs and therefore reserved.`

One way to fix this is by updating the task to pass those in as `op_kwargs` instead, the way it's done in the provider dag factory.

Reply:
I think this may be addressed by moving `task` into the kwargs anyway, which is something we spoke about above! Great call out.